src/pages/markets/components/helper/_available-platforms.tsx

```diff
@@ -94,9 +94,8 @@ const AvailablePlatforms = ({
     m_top,
 }: AvailablePlatformsProps) => {
     const { is_eu } = useRegion()
-    const deriv_mt5_name: String = 'Deriv MT5'
-    const deriv_go_name: String = 'Deriv GO'
```

Comment:
does TS have type `String`?
As far as I know we're using `TString` from utils for translation text